JavaScript/React.md

```diff
@@ -530,20 +603,37 @@ export class ComponentName extends React.Component {
   }
 
   render() {
-    return (
-      <div>{this.fullName}</div>
-    );
+    return <div>{this.fullName}</div>;
   }
 }
+
+export const ComponentName = observer(() => {
+  const state = useLocalStore(() => ({
+    firstName: 'John',
+    lastName: 'Doe',
+    get fullName() {
+      return `${state.firstName} ${state.lastName}`;
+    }
+  }));
+
+  return (
+    <div>
+      // ...
+      <div>{state.fullName}</div>
+    </div>
+  );
+});
 ```
 
 ### Actions
-Wrap methods and functions that modify state with *@action*. For binding action methods, use the aforementioned *@bind* decorator from the [`developit/decko`](https://github.com/developit/decko) library.
 
-``` jsx
+Wrap methods and functions that modify state with _@action_. For binding action methods, use the aforementioned _@bind_ decorator from the [`developit/decko`](https://github.com/developit/decko) library (for class components).
```

Comment:
I would go with `@action.bound` in that case...

Reply:
the reason we initially said no `@action.bound` is because we noticed people being lazy and using it even if the action was not needed.